src/NServiceBus.Serverless/Recoverability/ServerlessRecoverabilityPolicy.cs

```diff
@@ -23,6 +23,11 @@ public RecoverabilityAction Invoke(RecoverabilityConfig config, ErrorContext err
                 return RecoverabilityAction.ImmediateRetry();
             }
 
+            if (errorContext.DelayedDeliveriesPerformed <= config.Delayed.MaxNumberOfRetries)
+            {
+                return RecoverabilityAction.DelayedRetry(config.Delayed.TimeIncrease);
```

Comment:
the core does calculate the backoff time a bit differently. Not sure how similar we need to be though.

Reply:
@timbussmann have a look I simplified it